docs/integrations/optuna.rst

```diff
@@ -3,150 +3,214 @@
 Neptune-Optuna Integration
 ==========================
 
-This integration enables you to monitor |Optuna| hyperparameter optimization in Neptune.
+|Run on Colab|
 
-.. image:: ../_static/images/integrations/optuna_neptuneai.png
-   :target: ../_static/images/integrations/optuna_neptuneai.png
-   :alt: Optuna Neptune integration
+What will you get with this integration?
+----------------------------------------
 
-Requirements
-------------
-To use Neptune + Optuna integration you need to have installed is |neptune-client| and |neptune-contrib|.
+|optuna-tour|
 
-.. code-block:: bash
+|Optuna| is an open source hyperparameter optimization framework to automate hyperparameter search. With Neptune integration, you can:
 
-    pip install neptune-client neptune-contrib['monitoring']
+* see the experiment as it is running,
+* see charts of logged run scores,
+* log the parameters tried at every run,
+* log the best parameters after training,
+* explore interactive optuna visualizations like plot_contour, plot_slice, plot_parallel_coordinate, optimization_history,
+* save the study object.
+   
+.. note::
 
+    This integration is tested with ``optuna==2.3.0`` and current latest, ``neptune-client==0.4.125`` and current latest, and ``neptune-contrib==0.24.7`` and current latest.
 
-Initialize Neptune and create an experiment
--------------------------------------------
+Where to start?
+---------------
+To get started with this integration, follow the :ref:`Quickstart <quickstart>` below. You can also skip the basics and take a look at the :ref:`advanced options <advanced-options>`.
```

Comment:
I think it is safer to have the pointer <optuna-advanced-options> rather just advanced-options as the latter can be ambiguous.